client/src/components/Stars.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faStar as fasFaStar } from '@fortawesome/free-solid-svg-icons';
+import { faStar as farFaStar} from '@fortawesome/free-regular-svg-icons';
```

Comment:
`farFaStar` and `fasFaStar` are awfully similar variable names.. I'm glad they're only used once